python/ray/tune/syncer.py

```diff
@@ -103,7 +114,7 @@ def validate_sync_config(sync_config: "SyncConfig"):
         )
 
 
-def log_sync_template(options: str = ""):
+def get_rsync_template_if_available(options: str = ""):
```

Comment:
nit: can be removed?

Reply:
If rsync is available, we should probably continue to use it for same backwards behavior (and because it may be more efficient)?

Reply:
If we want to remove it, let's do it in a follow-up PR so we can easily revert

Reply:
Follow-up PR sounds good. The comment is just based on the observation that right now no caller is passing in this `options`.

Reply:
let's remove unused code.

Reply:
It is used at the moment though

Reply:
Right, the `options` are not used, but this was the case before this PR, so let's remove in follow up